django_mongodb_backend/schema.py

```diff
@@ -418,3 +419,46 @@ def _field_should_have_unique(self, field):
         db_type = field.db_type(self.connection)
         # The _id column is automatically unique.
         return db_type and field.unique and field.column != "_id"
+
+    def _create_collection(self, model):
+        """
+        If the model is encrypted create an encrypted collection with the
+        encrypted fields map else create a normal collection.
+        """
+
+        db = self.get_database()
+        if getattr(model, "encrypted", False):
+            client = self.connection.connection
+            ce = get_client_encryption(
+                client,
+                key_vault_namespace=settings.KEY_VAULT_NAMESPACE,
+                kms_providers=settings.KMS_PROVIDERS,
+            )
+            ce.create_encrypted_collection(
+                db,
+                model._meta.db_table,
+                self._get_encrypted_fields_map(model),
+                settings.KMS_PROVIDER,
```

Comment:
Is it intentional to have `KMS_PROVIDERS` and `KMS_PROVIDER`? It would be helpful to stub out some documentation (even just in a temporary example settings file) so that I can keep my local settings in sync as things evolve.

Incidentally, adding settings should be done cautiously. Django is often at war with settings proliferation.

Reply:
Yes, `KMS_PROVIDERS` is the dictionary of providers and `KMS_PROVIDER` is the configured provider. I started adding some configuration docs in e55763231f2cac49a813cf0dee04f60623306617

Reply:
Another setting = global state alert! Need to understand the use case of specifying a KMS provider. Is it per database, per model, etc? I would naively assume a use case for the latter since `ClientEncryption` needed a list of all providers whereas the call to create the collection uses a particular provider.

Reply:
bf26a8a10370217243ed33e0bdee6b0ef2a34d3b